WordPressAuthenticator/Authenticator/WordPressAuthenticatorStyles.swift

```diff
@@ -68,6 +68,10 @@ public struct WordPressAuthenticatorStyle {
 
     public let navBarBadgeColor: UIColor
 
+    public let navBarBackgroundColor: UIColor
+    
+    public let navButtonTextColor: UIColor
```

Comment:
These were added to the old Style struct so Auth knows how to style the nav bar when going from a new unified view to an old view.